src/library-authoring/common/messages.js

```diff
@@ -141,6 +141,16 @@ const messages = defineMessages({
     defaultMessage: 'PageOfCount',
     description: 'Label of the pagination item for page of count used for aria text',
   },
+  'library.common.footer.hyperlink.aboutLibraries': {
```

Comment:
It's better, not to use a camel case here.
`library.common.footer.hyperlink.about` - it should be enough

Reply:
done